### DISCUSSION TITLE
Is there any `consistent-type-imports` compatible rules?

### DISCUSSION THREAD

**Discussion by iyansr** [biomejs/biome#1180] [Q&A] [answered]
I saw `useGroupedTypeImport` but didn't match the behaviour i wanted from eslint's  `consistent-type-imports`

**Comment by ImBIOS** [comment, marked as answer]
- https://github.com/biomejs/biome/discussions/3
- https://github.com/biomejs/biome/pull/29#issuecomment-1839727486

you can find it here, the direct answer is "not yet".

**Reply by iyansr** [reply]
Thank you

**Comment by Conaclos** [comment]
I am working on this. This should be ready by the end of this week :)

**Reply by ImBIOS** [reply]
Wow, another happy news! 👍

**Reply by iyansr** [reply]
Thank you
